### ISSUE TITLE
(create-zephyr-app) Prompt choice between yaml and json

### ISSUE THREAD

**Issue by marvin-j97** [zephyr-js/zephyr#4]
I personally quite dislike yaml, so the first thing I'd always do after using `create-zephyr-app` would be converting those back to json.
It would be nice to be able to choose between json and yaml, and write `prettierrc.yml` and `eslintrc.yml` on demand using JSON or YAML (https://www.npmjs.com/package/yaml).
It would also help in the future for modifying those configs (like choosing different rule sets if the user wants to).

**Comment by kakengloh** [comment]
Yeah we should prompt user for some config preference, planning to refactor `create-zephyr-app` soon. I think [`create-astro`](https://github.com/withastro/astro/tree/main/packages/create-astro) is really great, might be a good reference.
